src/App.js

```diff
@@ -54,7 +70,10 @@ class App extends React.Component {
       ? instances.find(({ id }) => id === instanceId)
       : instances[0]
 
-    this.changePath(`/${appId}${instance ? `/${instance.id}` : ''}`)
+    this.changePath(
+      `/${appId}${instance ? `/${instance.id}` : ''}`,
+      params ? `?params=${params}` : ''
```

Comment:
The idea here is to pass a serialized object rather than a query string, so that from the app, we only send / receive data objects with the wrapper. Note that the implementation will need to be adapted in order to support iframe apps.

The reason why I chose a single parameter, rather than converting each string to a query string parameter, is to allow the addition of other query string params, that are not related to the app instance (e.g. `?debug=1`).